### ISSUE TITLE
Add preview API support

### ISSUE THREAD

**Issue by otbe** [contentful-labs/cf-graphql#9]
Hi,

I saw your v0.3 PR which adds CMA for schema generation. 
Is there any reason why there is no support for CPA? Would love to use it for testing new models or new content.

Thought of a small lambda via serverless :)

**Comment by jelz** [comment]
Hi there,

the problem is more complex:

1. I have to use CMA for v0.3 because not all validations are exposed on CDA responses for the content types endpoint. There's a bug caused by using CDA for CT fetching (the one you've opened #8  and #6 ). But this is the only place where we need CMA.
2. You never query CMA for entry/asset data. The library always uses CDA to get the data
3. I could provide a flag to switch between CDA and CPA, but I don't want to make it a part of v0.3

To sum up:
- v0.3 will make bugs go away
- v0.4 will allow you to switch between CDA and CPA

Additionally, v0.4 is going to include two deployment methods:
- AWS API Gateway + Lambda
- [Zeit's now](https://zeit.co/now)

**Comment by otbe** [comment]
I think the problem is really simple. CMA for schema generation and CDA/CPA for fetching data :)
Great sum up!

Nice plans for v0.4, I guess this won't keeping me off to deploy/code it by my self? I mean its stupid simple to deploy your example via serverless to AWS Lambda or run it on bare metal right now. :)

**Comment by jelz** [comment]
Both CPA support and deployment to AWS Lambda were moved to v0.5 release.

**Comment by jelz** [comment]
Released in v0.5
